### DISCUSSION TITLE
Initial Status Bar color in Android

### DISCUSSION THREAD

**Discussion by Nasicus** [ionic-team/capacitor#3162] [Q&A]
So we probably all know we can change the status bar color in Android via:
https://capacitor.ionicframework.com/docs/apis/status-bar/

However when I do this in my JS code I'm too late and initially there is a different color:
![image](https://user-images.githubusercontent.com/1086065/85917531-10b42e80-b85b-11ea-9831-f96f33b954e6.png)

Is there any way to set the initial value?

I tried changing something in `app\src\main\res\values\styles.xml` that didn't help.

Any help is appreciated!

**Comment by piotr-cz** [comment]
Try with
```xml
<?xml version="1.0" encoding="utf-8"?>
<resources>
    <style name="AppTheme.NoActionBar" parent="Theme.AppCompat.DayNight.NoActionBar">
        <item name="android:windowLightStatusBar">true</item>
        <item name="android:statusBarColor">#ff0000</item>
    </style>
</resources>
```
